go.mod

```diff
@@ -77,6 +77,7 @@ require (
 	github.com/urfave/cli/v2 v2.3.0
 	github.com/wealdtech/go-bytesutil v1.1.1
 	github.com/wealdtech/go-eth2-util v1.6.3
+	//github.com/MariusVanDerWijden/tx-fuzz v0.0.0-20220222104514-79320b80893a
```

Comment:
Basically I need to have [this](https://github.com/MariusVanDerWijden/tx-fuzz/pull/3) merged in otherwise go.mod goes crazy. So i am commenting it out for now